tasks/team-access.yaml

```diff
@@ -56,32 +42,80 @@
     status_code: 200
   register: access_scope_result
 
-# Create all of the roles
+# Create all of the roles, create_role is depend on access_scope_result
 - name: Create Roles
   include_tasks: create_role.yaml
   with_items: "{{ roles }}"
 
-# Now we need to map
+# Retrieve the auth provider ID, auth has been created in CCM so we can just retrive here.
+- name: Get Auth Provider Id
+  uri:
+    # auth_provider.name: OpenShift in vars
+    url: "https://{{ api_endpoint }}/v1/authProviders?name=OpenShift"
+    method: GET
+    validate_certs: no
+    headers:
+      authorization: "{{ authorization }}"
+    body_format: json
+    status_code: 200
+  register: authprovider_result
+
+- name: Keep the auth provider ID for later use
+  set_fact:
+    "auth_provider_openshift_id": "{{ authprovider_result.json.authProviders[0].id }}"
+
+- name: Declare role name that will use cross the script
```

Comment:
role name is set in `vars/acs-team-access-config.yaml` already

Reply:
That is in an array and has to stay in array for create-role.yaml. And makes it wired to reuse in this component